product_expiry_simple/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# © 2018 Jarsa Sistemas (Alan Ramos <alan.ramos@jarsa.com.mx>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Expiry Simple',
+    'version': '11.0.1.0.0',
+    'category': 'Product',
+    'license': 'AGPL-3',
+    'summary':
+    'Simpler and better alternative to the official product_expiry module',
+    'author': 'Akretion,Jarsa Sistemas,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
```

Comment:
@alan196 Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)